lib/entities/workflows-changelog-entry.ts

```diff
@@ -0,0 +1,63 @@
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import copy from 'fast-copy'
+import {
+  DefaultElements,
+  Link,
+  MakeRequest,
+  PaginationQueryOptions,
+  SysLink,
+} from '../common-types'
+import { wrapCollection } from '../common-utils'
+import enhanceWithMethods from '../enhance-with-methods'
+
+export type WorkflowsChangelogQueryOptions = Omit<PaginationQueryOptions, 'order'> & {
```

Comment:
This has unfortunately changed a bit after the latest discussion I had with @aromanko. For now we agreed on this final shape:

```
export type WorkflowsChangelogQueryOptions = Omit<PaginationQueryOptions, 'order'> & {
  'entity.sys.linkType': string
  'entity.sys.id': string
  'workflow.sys.id': string
}
```

i.e. entityId, entityType and workflowId all required params for this first iteration.

Reply:
You mean @ronaldronson ? :P

Reply:
> You mean @ronaldronson ? :P

damn yes 😆 always tricky to find him

Reply:
https://media1.giphy.com/media/l0Iyl55kTeh71nTXy/giphy.gif?cid=ecf05e47k7d9a2d1sdktvhknh9gbwbbwm25ymlh07zzwrxlf&rid=giphy.gif&ct=g